packages/libs/eda/src/lib/map/analysis/DraggableVisualization.tsx

```diff
@@ -71,7 +76,11 @@ export default function DraggableVisualization({
       confineToParentContainer
       showPanelTitle
       isOpen
-      styleOverrides={{ zIndex: zIndexForStackingContext, resize: 'both' }}
+      styleOverrides={{
+        zIndex: zIndexForStackingContext,
+        resize: 'both',
+        overflow: 'hidden',
```

Comment:
tbh I don't fully understand the `overflow` property usage here.
It may be a browser difference (though I've tried Chrome and Firefox).
I am seeing scroll bars appear on mouse-over. Perhaps you could explain to me later? Sorry!

Reply:
We want it to scroll, but we don't want `DraggablePanel` to handle the scrolling; we want to handle it ourselves in the divs we pass as children of `DraggablePanel`. Thus you'll see `overflow: 'auto'` on one of the child divs below.